google-cloud-contrib/google-cloud-logging-logback/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>google-cloud-logging-logback</artifactId>
+    <version>0.17.3-alpha-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>Google Cloud Logging Logback Appender</name>
+    <url>https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-contrib/google-cloud-logging-logback</url>
+    <description>
+        Logback support for Google Cloud Logging
+    </description>
+
+    <properties>
+        <logback.version>1.2.3</logback.version>
+    </properties>
+
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-contrib</artifactId>
+        <version>0.17.3-alpha-SNAPSHOT</version>
+    </parent>
+    <dependencies>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>${logback.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-logging</artifactId>
+        </dependency>
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
switched to easymock. I've had to extend the `LoggingAppender` class  to override `getLogging` and `getProjectId` : could n't figure out a way with partial mock, if there is an easier way, please let me know.